front/src/applications/stdcm/views/StdcmView.tsx

```diff
@@ -57,7 +57,6 @@ const StdcmView = () => {
   const handleSelectSimulation = (index: number) => {
     if (retainedSimulationIndex === -1) {
       setSelectedSimulationIndex(index);
-      setShowBtnToLaunchSimulation(false);
```

Comment:
Also here, I'm not sure what it has to do with this PR.

Moreover, when the user switches in a simulation, he won't want to relaunch the exact same one. That's why the launch button was hidden I suppose. It will be displayed again when the user changes a param. I think we should leave it that way.

Reply:
No problem, I'm removing it 😅